setup.py

```diff
@@ -8,7 +8,7 @@
     'flake8',
     'pytest==7.0.1',
     'pytest-mock>=3.5.1',
-    'coverage==6.2',
+    'coverage==6.1',
```

Comment:
why do you downgrade this dependency?

Reply:
The tests were failing, I will change the version back to 6.2